platform/ios/test/MGLAnnotationViewTests.m

```diff
@@ -0,0 +1,62 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+static NSString * const MGLTestAnnotationReuseIdentifer = @"MGLTestAnnotationReuseIdentifer";
+
+@interface MGLTestAnnotation : NSObject <MGLAnnotation>
+@property (nonatomic, assign) CLLocationCoordinate2D coordinate;
+@end
+
+@implementation MGLTestAnnotation
+@end
+
+@interface MGLAnnotationViewTests : XCTestCase <MGLMapViewDelegate>
+@property (nonatomic) XCTestExpectation *expectation;
+@property (nonatomic) MGLMapView *mapView;
+@end
+
+@implementation MGLAnnotationViewTests
+
+- (void)testAnnotationView
+{
+    _mapView = [[MGLMapView alloc] initWithFrame:CGRectZero];
+    _mapView.delegate = self;
+    
+    _expectation = [self expectationWithDescription:@"test annotation property"];
+    
+    MGLTestAnnotation *annotation = [MGLTestAnnotation new];
```

Comment:
But: use alloc/init rather than new. In general, using init directly makes it easier to add parameters to an initializer. (Admittedly, it used to matter more in the days before ARC.) In this codebase specifically, where many of the files are Objective-C++, we prefer to leave the new keyword to C++ usage.